packages/atomic/src/components/atomic-layout/atomic-layout.pcss

```diff
@@ -0,0 +1,27 @@
+@import './search-layout.pcss';
```

Comment:
Here is all the CSS that is "static" meaning that does not need to adapt whether some sections are present are not.
In our current search layout, 3 things are dynamic:
- the facets, we don't display an empty column if they are not defined. In the case where some facets are defined but return no results, I think it's better to keep the column not to have a jittery interface
- the mobile media query, we can't use a CSS variable for it unfortunately, which is why it's dynamically generated
- the refine toggle/modal which hides the sort dropdown when in mobile view

Reply:
alternative is to make the no results image take both columns, but the search box shouldn't move